project/Mima.scala

```diff
@@ -30,6 +30,8 @@ object Mima {
     ProblemFilters.exclude[MissingClassProblem]("scala.meta.testkit.DiffAssertions"),
     ProblemFilters.exclude[MissingClassProblem]("scala.meta.testkit.DiffAssertions$DiffFailure"),
     ProblemFilters.exclude[MissingClassProblem]("scala.meta.testkit.DiffAssertions$DiffFailure$"),
-    ProblemFilters.exclude[MissingTypesProblem]("scala.meta.testkit.DiffAssertions")
+    ProblemFilters.exclude[MissingTypesProblem]("scala.meta.testkit.DiffAssertions"),
+    ProblemFilters.exclude[ReversedMissingMethodProblem]("scala.meta.Template.getDerives"),
```

Comment:
Template trait has these added and if someone did extend Template it could be an issue, however it is very unlikely that someone does it